.github/workflows/plugins_react_tests.yml

```
@@ -0,0 +1,71 @@
+name: (non-blocking) React on plugins
+
+on:
+  pull_request:
+    branches: [develop]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    # since it's a non-blocking test, if one plugin tests are failing,
+    # it shouldn't prevent the rest of the plugins tests from running.
+    continue-on-error: true
+    steps:
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      # We could update the postinstall action for foreman to look for an environment variable for plugin webpack dirs
+      # before kicking off the ruby script to find them, this would eliminate the ruby dep and running `npm i` in Katello.
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.6
+      - name: Checkout Foreman
+        uses: actions/checkout@v2
+        with:
+          path: ./projects/foreman
+      - name: Install Foreman npm dependencies
+        run: npm i
```

I'd always use long options in scripts
```suggestion
        run: npm install
```